test/gtest/uct/uct_test.cc

```diff
@@ -1625,16 +1625,32 @@ void test_uct_iface_attrs::basic_iov_test()
 
     EXPECT_FALSE(max_iov_map.empty());
 
-    if (max_iov_map.find("am")  != max_iov_map.end()) {
+    if (m_e->iface_attr().cap.flags & (UCT_IFACE_FLAG_AM_SHORT |
+                                       UCT_IFACE_FLAG_AM_BCOPY |
+                                       UCT_IFACE_FLAG_AM_ZCOPY)) {
+        EXPECT_NE(max_iov_map.end(), max_iov_map.find("am"));
```

Comment:
alternatively you can check the presence:
`EXPECT_EQ(map.count("key"), 1)`

Reply:
why is it better? we are testing presence but not amount, IMO `find` is better here also this check was in initial code

Reply:
Essentially in map count also checks the presence, because the returned value can vary between 0 and 1.
For the the only pros is that it's shorter and looks more readable, but up to you